### PR TITLE
Feature category

### PR DESCRIPTION
**Pull request by NamrataRaikwar2002**
-user can redirect to the selected category by clicking on the feature category from home page.
 

preview:https://sky-mart-app2022-7f7bkdl79-namrataraikwar2002.vercel.app/

### CODE REVIEW

File: src/pages/Authentication/Signup.jsx [hunk 1]
```diff
@@ -18,51 +20,100 @@ export const Signup = () => {
...
+                className="login_input passwordInputDiv"
+                placeholder="Reenter Password"
+                id="confirmPswdinput"
+                required
+              />
+              {
+                <div
+                  onClick={() =>
+                    inputType.confirmpaswd === 'text'
+                      ? setinputType({ ...inputType, confirmpaswd: 'password' })
```
Comment (KoushikKintali) [inline src/pages/Authentication/Signup.jsx:85]: it looks clean if the function inside onclick is moved as a separate function

File: src/pages/Cart/Cart.jsx [hunk 2]
```diff
@@ -1,61 +1,77 @@
...
         <section className="cart_management content">
+            <h2 className='productHeading'>MY CART({cartList.length})</h2>
+                {cartList.length >=1 ? 
+                <>
             <div className="section_background">
-                <h2>MY CART(1)</h2>
+                <div className='cartListDiv'>
+                   <CartCard />
+                </div>
+                {/* <div className='priceImgDiv'> */}
```
Comment (KoushikKintali) [inline src/pages/Cart/Cart.jsx:30]: please remove the commented code if not needed

File: src/pages/Cart/component/CartCard.jsx [hunk 3]
```diff
@@ -1,36 +1,97 @@
...
+                        payload: { _id: _id },
+                      })
+                    }
+                  >
+                    Remove From Cart
+                  </button>
+                  <button
+                    className="e_com_btn wishlist_btn"
+                    onClick={() =>
+                      productDispatch({
```
Comment (KoushikKintali) [inline src/pages/Cart/component/CartCard.jsx:73]: u can move productDispatch in separate function & call the function here

**Review by KoushikKintali — CHANGES_REQUESTED** [review]
Please check the comments

**Review by Tanjum786 — APPROVED** [review]
code look clean and understandable
well structured good work good to go

**Pull request merged**